portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,10 +39,12 @@
   private class Comment{
     String comment;
     String name;
+    long timestamp;
 
-    public Comment(String c, String n){
+    public Comment(String c, String n, long timestamp){
         comment = c;
         name = n;
+        timestamp = timestamp;
```

Comment:
this doesn't do what you think it does -- timestamp (the function parameter variable) is assigned to itself.

This is why it's very common to see:

```
 this.timestamp = timestamp;
```

That fully qualifies the local field and allows reusing the field name as a constructor parameter.